services/ui-src/src/components/fields/DropdownField.tsx

```diff
@@ -46,11 +46,20 @@ export const DropdownField = ({
   const formatOptions = (options: DropdownOptions[] | string) => {
     let dropdownOptions = [];
     if (options === "copyEligibleReports") {
-      dropdownOptions =
-        submittedReportsByState?.map((option) => ({
-          label: `${option.programName} ${convertDateUtcToEt(option.dueDate)}`,
-          value: option.fieldDataId!,
-        })) ?? [];
+      if (submittedReportsByState?.length == 0) {
+        dropdownOptions.push({
+          label: dropdownNoReports,
+          value: "",
+        });
+      } else {
+        dropdownOptions =
+          submittedReportsByState?.map((option) => ({
+            label: `${option.programName} ${convertDateUtcToEt(
+              option.dueDate
+            )}`,
+            value: option.fieldDataId!,
+          })) ?? [];
```

Comment:
Request: A typescript warning about `fieldDataId` was being overridden by the `!` on line 60, but this is unnecessary because according to lines 106 & 173 in `services/app-api/handlers/reports/create.ts`, all reports will always have a `fieldDataId`. However, our current types for `ReportMetadataShape` show this key/value pair as optional. 

Please update both ui-src/frontend type ([here](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/blob/a7564724410d67194d38ee383feca29cf1db4021/services/app-api/utils/types/reportContext.ts#L12)) and app-api/backend type ([here](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/blob/a7564724410d67194d38ee383feca29cf1db4021/services/app-api/utils/types/reportContext.ts#L12)) to have a non-optional `fieldDataId: string`.